frontend/VolunteerOne/screens/Onboarding/NewPassword.js

```diff
@@ -15,136 +15,82 @@ import { Block, Text } from "galio-framework";
-  const [email, setEmail] = useState("");
-  const [password, setPassword] = useState("");
+  const [newPassword, setNewPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
 
-  function handleEmailInput(input) {
-    setEmail(input);
-  }
+  const handleNewPasswordInput = (input) => {
+    setNewPassword(input);
+  };
 
-  function handlePasswordInput(input) {
-    setPassword(input);
-  }
+  const handleConfirmPasswordInput = (input) => {
+    setConfirmPassword(input);
+  };
 
-  function handleLoginBtnClick() {
-    console.log(email, password);
-    navigation.navigate("App");
-  }
+  const handleSaveBtnClick = () => {
+    console.log(newPassword, confirmPassword);
+    navigation.navigate('Login');
+  };
 
-  return (
+return (
     <Block flex middle>
       <StatusBar hidden />
       <ImageBackground
         source={Images.RegisterBackground}
         style={{ width, height, zIndex: 1 }}
       >
-        <Block safe flex middle>
-          <Block style={styles.loginContainer}>
-            <Block flex>
-              <Block flex={0.5} middle style={styles.instructionText}>
-                <Image source={logo} />
-              </Block>
-              <Block flex={0.17} middle style={styles.instructionText}>
-                <TouchableOpacity
-                  onPress={() => console.log("create account btn")}
-                >
-                  <Text
-                    color="#8898AA"
-                    size={12}
-                    style={{
-                      fontWeight: "bold",
-                      textDecorationLine: "underline",
-                      paddingRight: 5,
-                    }}
-                  >
-                    Create Account
-                  </Text>
-                </TouchableOpacity>
-                <Text color="#8898AA" size={12}>
-                  or Login with credentials
-                </Text>
+      <Block safe flex middle>
+      <Block style={styles.container}>
+      <Block flex={0.5} middle>
+                <Block center>
+                  <Image source={Images.VolunteerOneIcon} style={styles.logo} />
+                </Block>
               </Block>
-              <Block flex center>
-                <KeyboardAvoidingView
-                  style={{ flex: 1 }}
-                  behavior="padding"
-                  enabled
-                >
-                  <Block width={width * 0.8} style={{ marginBottom: 15 }}>
-                    <TextInput
-                      style={styles.input}
-                      placeholder="Email"
-                      onChangeText={handleEmailInput}
-                    />
-                  </Block>
-                  <Block width={width * 0.8}>
-                    <TextInput
-                      secureTextEntry={true}
-                      style={styles.input}
-                      placeholder="Password"
-                      onChangeText={handlePasswordInput}
-                    />
```

Comment:
You can probably copy this over and change some variables names to fix the styling